bioptim/dynamics/ode_solver.py

```diff
@@ -146,6 +146,38 @@ class OdeSolver:
     The public interface to the different OdeSolvers
     """
 
+    class RK1(RK):
+        """
+        A Runge-Kutta 1 solver (Forward Euler Method)
+        """
+
+        def __init__(self, n_integration_steps: int = 5):
```

Comment:
Is 5 as default makes sense?

Reply:
I think so because every runge-kutta integrators have a 5 step by default. otherwise if one I would rise a warning to aware the user